apps/codeforafrica/src/components/StayInTouch/StayInTouch.js

```diff
@@ -59,22 +59,24 @@ const StayInTouch = React.forwardRef(function StayInTouch(
         {links.map((media) => (
           <Grid
             item
-            key={media.href}
+            key={media.id}
             sx={{
               pr: "10px",
               ":last-of-type": {
                 pr: 0,
               },
             }}
           >
-            <Link
-              sx={{
-                display: "block",
-              }}
-              href={media.href}
-            >
-              <IconRoot src={media.icon.src} alt={media.label} />
-            </Link>
+            {media?.url ? (
```

Comment:
Put this check around the `Grid item`. Why add a `Grid item` if there is no `Link`?